gme/Blip_Buffer.h

```diff
@@ -121,8 +121,6 @@ class Blip_Buffer {
 	friend class Blip_Reader;
 };
 
-#include "blargg_config.h"
```

Comment:
Why remove this?

It can be useful when building through standalone makefiles, for e.g.

Reply:
Okay, it's been put back in.